salt/states/zabbix_host.py

```diff
@@ -300,11 +300,14 @@ def _interface_format(interfaces_data):
         if host_updated_params:
             update_host = True
 
-        host_inventory_mode = host["inventory_mode"]
-        inventory_mode = host_extra_properties.get(
-            "inventory_mode",
-            "0" if host_inventory_mode == "-1" else host_inventory_mode,
-        )
+        try:
+            host_inventory_mode = host["inventory_mode"]
```

Comment:
Wouldn't it be simpler to use the following?:
```suggestion
            host_inventory_mode = host.get("inventory_mode", None)
```
Then you wouldn't need the whole try/except block